docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: '3.3'
+
+services:
+  core:
+    # image: tokend/core:3.2.1
+    image: registry.gitlab.com/tokend/core:pg-b5c157b9a8377008bcc791492536e5dea59d2845
```

Comment:
fix

Reply:
We must prepare tokend/core:x.x.x release with built-in curl
I've created merge request in core #95